src/api/embedsApi.ts

```diff
@@ -290,6 +291,11 @@ const campaignBlockMeta: Fetch<CampaignBlockMetaData> = async ({ embedData, cont
   return { image };
 };
 
+const uuDisclaimerMeta: Fetch<UuDisclaimerMetaData> = async ({ embedData, context}) => {
+  const article = embedData.articleId ? await fetchSimpleArticle(`urn:article:${embedData.articleId}`, context) : undefined;
+  return article ? { disclaimerLink: { text: article.title.title, href: `/article/${article.id}`}} : {} ;
```

Comment:
Måtte returnere {} som default for at ikkje https://github.com/NDLANO/frontend-packages/commit/2a2314f1016ca54c4f4feea0d15a9f79d1b32198 skulle knekke koden.